clmm/theory/ccl.py

```diff
@@ -105,6 +102,9 @@ def _get_mass(self):
     def _set_concentration(self, cdelta):
         """"set concentration. Also sets/updates hdpm"""
         self.conc = ccl.halos.ConcentrationConstant(c=cdelta, mdef=self.mdef)
+        # adjust it for ccl version > 2.7.0
+        if parse(ccl.__version__) > parse('2.7.0'):
+            ccl.UnlockInstance.Funlock(type(self.mdef), "_concentration_init", True)
```

Comment:
Here we do not support CCL >= 2.7.1.dev10 because we need change many places to make it work, not only this part. Do we want to write a comment about this fact explicitly?